cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -28,7 +28,13 @@
   {% endif -%}
 {% endif -%}
 
-{% set params = "--master=127.0.0.1:8080" + " " + cluster_name + " " + cluster_cidr + " " + allocate_node_cidrs + " " + cloud_provider  + " " + cloud_config + service_account_key + pillar['log_level'] -%}
+{% set root_ca_file = "" -%}
+
+{% if grains['cloud'] is defined and grains.cloud in [ 'aws', 'gce' ]    %}
```

Comment:
I'm following the example [here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/saltbase/salt/kube-apiserver/kube-apiserver.manifest#L55-L57). Incorrectly adding the flag will break a cloud provider so I want to err on the side of caution. I'll keep an eye on #10236